Rubberduck.Parsing/VBA/IBuiltInDeclarationLoader.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace Rubberduck.Parsing.VBA
+{
+    public interface IBuiltInDeclarationLoader
```

Comment:
I actually hope the `BuiltInDeclarationLoader` can die at some point when the `COMReferenceLoader` is able to provide the aliases, debug declarations and the hidden forms stuff we still load via the custom declaration loaders. This interface and the corresponding implementation has been introduced to have a clean way of reporting whether the last run of the custom declaration loaders actually yielded something. We need that information to avoid refreshing `DeclarationFinder` every time after loading the references even when we do not need to.